source/_docs/disaster-recovery.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Early Access: Site Disaster Recovery
+description: Learn how mission-critical websites can stay online in the event of a total zone failure
+earlyaccess: true
+earlynote: This documentation covers features and options not available across the entire platform.
+---
+
+## Access
+
+Site Disaster Recovery (DR) is available for purchase as an add-on to all Elite site plans. It's not available for Elite Starter plans. For more information, please [contact Sales](https://pantheon.io/contact-us).
```

Comment:
Are "Elite Starter plans" not considered "Elite site plans"?

Reply:
yeah, do we need an "except" or "however" in there?